labels/docker.io+anchore+test_images@sha256:10008791acbc5866de04108746a02a0c4029ce3a4400a9b3dad45d7f2245f9da/cf8d4a74-dcae-46a4-b152-36491b09f8bf.json

```diff
@@ -0,0 +1 @@
+{"ID": "cf8d4a74-dcae-46a4-b152-36491b09f8bf", "effective_cve": "CVE-2019-1003092", "image": {"exact": "docker.io/anchore/test_images@sha256:10008791acbc5866de04108746a02a0c4029ce3a4400a9b3dad45d7f2245f9da"}, "label": "FP", "package": {"name": "nomad", "version": "0.7.4"}, "timestamp": "2023-08-04T17:12:23.021783", "tool": "grype@v0.63.1+profile=cpe", "user": "bress", "vulnerability_id": "CVE-2019-1003092"}
```

Comment:
I think this one may be a TP if this is the nomad jenkins plugin? https://github.com/CVEProject/cvelist/blob/9e6221f0212e3a5e5d5279b8b0a2ad478baf4f3e/2019/1003xxx/CVE-2019-1003092.json#L13-L29 says all versions are vulnerable, but GitHub doesn't have this one labeled yet: https://github.com/advisories/GHSA-5q63-jvc9-qphv

Reply:
I didn't look at the actual results though to see if this is the go package or the jenkins plugin, so ignore me if it's the go module

Reply:
I submitted a PR to fix the GitHub entry
https://github.com/github/advisory-database/pull/2590

It's fixed in version 0.5.1

I'm pleased you looked at this, thanks!

Reply:
Ah great, thanks!  They just merged all of mine!